fe/src/main/java/org/apache/doris/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -59,21 +59,23 @@ public KafkaRoutineLoadJob(long id, String name, String userName, long dbId, lon
         this.topic = topic;
     }
 
+    public KafkaProgress getProgress() {
+        Gson gson = new Gson();
+        return gson.fromJson(this.progress, KafkaProgress.class);
+    }
+
     @Override
-    public List<RoutineLoadTask> divideRoutineLoadJob(int currentConcurrentTaskNum) {
+    public List<RoutineLoadTaskInfo> divideRoutineLoadJob(int currentConcurrentTaskNum) {
         // divide kafkaPartitions into tasks
-        List<KafkaRoutineLoadTask> kafkaRoutineLoadTaskList = new ArrayList<>();
+        List<RoutineLoadTaskInfo> kafkaRoutineLoadTaskList = new ArrayList<>();
         for (int i = 0; i < currentConcurrentTaskNum; i++) {
-            // TODO(ml): init load task
-            kafkaRoutineLoadTaskList.add(new KafkaRoutineLoadTask(getResourceInfo(), 0L, TTaskType.PUSH,
-                    dbId, tableId, 0L, 0L, 0L, SystemIdGenerator.getNextId()));
+            kafkaRoutineLoadTaskList.add(new KafkaTaskInfo(SystemIdGenerator.getNextId()));
```

Comment:
SystemIdGenerator will produce repetitive id if Frontend restart or Master FE changed.
Just use a UUID or random Long is better, if you don't want to persist this info.

Reply:
I think maybe I can reuse `CatalogIdGenerator`, but `CatalogIdGenerator` need add a field named `name`.